packages/providers/upload-aws-s3/README.md

```diff
@@ -64,6 +64,41 @@ module.exports = ({ env }) => ({
 });
 ```
 
+### Configuration for a private S3 bucket
+
+If your bucket is configured to be private, you will need to set the `ACL` option to `private` in the `params` object. This will ensure that the signed URL is generated with the correct permissions.
+
+You can also define the expiration time of the signed URL by setting the `signedUrlExpires` option in the `params` object. The default value is 7 days.
```

Comment:
Hello 🙃 Since the url is created each time the page is loaded, would it make sens to put a lower default value? Could 1 hour be enough?

Reply:
yep, that is a good point, we will change it to a lower value :)

Reply:
and now get back to your martini

Reply:
I'm gone
![Ru9sjtZ09XOEg](https://user-images.githubusercontent.com/11708220/222194768-261ffe55-91f5-48aa-b50c-95b14fb0b9cd.gif)